TeamShiksha/logoexecutive
# Analytics component implementation

### Summary

This PR introduces analytics section in admin dashboard

### Approach

Created a responsive analytics component according to figma design.
### How to Test the Changes

To test the responsiveness, make sure to remove the CSS for the root tag in index.css and add Analytics component in App.jsx.
Mobile View
![analyticsMobile](https://github.com/user-attachments/assets/2e9ce583-6897-421a-90da-a00fadeec309)

Desktop View
![analyticsDesktop](https://github.com/user-attachments/assets/2e55f932-25de-405e-b3db-99a4204996c4)

### Screenshots or Recordings



## Checklist
